scripts/ec2/networking.sh

```diff
@@ -29,13 +29,13 @@ readonly EC2_FILES='/var/tmp/ec2'
 
 [[ -d $EC2_FILES ]] || mkdir -p $EC2_FILES
 
-# The version 3.1.2 is currently the recommended version.
-SRIOV_DRIVER='ixgbevf-3.1.2.tar.gz'
+# The version 4.2.1 is currently the recommended version.
+SRIOV_DRIVER='ixgbevf-4.2.1.tar.gz'
 if [[ -n $SRIOV_DRIVER_VERSION ]]; then
     SRIOV_DRIVER="ixgbevf-${SRIOV_DRIVER_VERSION}.tar.gz"
 fi
 
-ENA_DRIVER='ena_linux_1.1.3.tar.gz'
+ENA_DRIVER='ena_linux_1.2.0.tar.gz'
```

Comment:
👍 

I saw them releasing it recently. I hope it also fixed compilation issues on 4.11+ kernels.